include/admin_navigation.js

```diff
@@ -235,9 +271,9 @@ module.exports = function AdminNavigationModule(pb) {
      * @method getAdditions
      * @returns {Array}
      */
-    function getAdditions() {
-        return util.clone(AdminNavigation.additions);
-    }
+    function getAdditions(site) {
+        return getAdditionsInScope(AdminNavigation.additions, site);
+    };
```

Comment:
Done.

Ben Tidwell
Software Engineer III | Talent Network
CareerBuilder
5550-A Peachtree Pkwy, Norcross GA 30092

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 9, 2015 at 5:53 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In include/admin_navigation.js<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34308304>:

> @@ -235,9 +271,9 @@ module.exports = function AdminNavigationModule(pb) {
>       * @method getAdditions
>       * @returns {Array}
>       */
> -    function getAdditions() {
> -        return util.clone(AdminNavigation.additions);
> -    }
> +    function getAdditions(site) {
> +        return getAdditionsInScope(AdminNavigation.additions, site);
> +    };


only assigned functions need semi-colons

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34308304>.